### DISCUSSION TITLE
Black PCBs ready to ship to you...USA address only

### DISCUSSION THREAD

**Discussion by caschy** [FundamentalFrequency/.github#30] [General]
Hi have 3 black PCBs ready for shipping now to USA address only.  Shipping via USPS Priority. Just pay for shipping. $10.61 paypal.  Reply if you are interested.  I'm also on the discord channel.

**Comment by astn7278** [comment]
Hey I'd love one if you still have any available!

**Reply by sramey** [reply]
I should have looked here before I ordered 5 to be printed haha.

**Reply by wawezz** [reply]
Hey I need one or two please)

**Comment by caschy** [comment]
Yeah I have one.  I can take paypal.

**Reply by astn7278** [reply]
Sounds good, let me know your email or PayPal name

**Reply by caschy** [reply]
can you msg me on the discord channel?  I'd prefer to not give out my info on here.

**Reply by ceraniciv** [reply]
I would like a PCB. Where can I locate the link for the discord channel to follow up? Thanks

**Reply by caschy** [reply]
https://discord.gg/aMsTJezv

try that link

**Comment by caschy** [comment]
2 left!

**Reply by caschy** [reply]
1 left now

**Reply by caprisun08** [reply]
ill take it! I can do paypal

**Reply by caprisun08** [reply]
I am eagon#9223 on discord

**Reply by caschy** [reply]
ALL GONE!

**Reply by n0ughtchill** [reply]
Dang just missed!

**Comment by wawezz** [comment]
Hey I steel need board ) can you provide info where you order your boards and with what params? @caschy . I was trying to make order on pcbway.com but I have difficulties with options need to check )) please help 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/34684084/177006052-c48f6510-7759-444f-a5fa-f838f528d405.png">

**Reply by caschy** [reply]
I used jlcpcb and used whatever default settings came up.

**Comment by MMDav** [comment]
Hi! Do you have any left?

**Reply by caschy** [reply]
No I have no more left

**Comment by briwil** [comment]
> Hi have 3 black PCBs ready for shipping now to USA address only.  Shipping via USPS Priority. Just pay for shipping. $10.61 paypal.  Reply if you are interested.  I'm also on the discord channel.

I wish I'd seen this earlier, I believe I paid $30 for mine from Gerber Labs

**Reply by caprisun08** [reply]
yes pls!
